bin/cask

```diff
@@ -236,26 +236,45 @@ def find_best_emacs():
     else:
         return 'emacs'
 
+def inside_emacs_24():
+    """Return true if we are running inside Emacs-24.
+
+Emacs 24 is rather inconsistent in its use of the EMACS and INSIDE_EMACS
```

Comment:
`$EMACS` and `$INSIDE_EMACS`, please, and also use backticks to mark both as literals.